system/security/shared/caching.hpp

```diff
@@ -151,6 +151,7 @@ class CPermissionsCache
 
     virtual ~CPermissionsCache();
 
+    //Returns a shared cache
     static CPermissionsCache* queryInstance()
```

Comment:
With the introduction of the secmgrbased cache, we should force the caller to use the new method.
Having the two methods could lead to confusion and misuse. Continue to support the case where no name is provided in the new method.